components/member-list-item/member-list-item.module.scss

```diff
@@ -5,11 +5,17 @@
   margin: 20px;
   border: 1px solid #f4f4f4;
   padding: 20px 10px;
+  background: white;
 
   //addition of box-shadow and border radius to member card
   box-shadow: 10px 10px 20px -10px rgba(0, 0, 0, 0.22);
   border-radius: 5%;
 
+  &:hover {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
good to have `-webkit-box-shadow:` and `-moz-box-shadow:`

Reply:
good addition (y)

Reply:
@Shihab-Duzno @Harshi7016 Always check: https://caniuse.com/#search=box-shadow before adding prefixes. Usually build tools like autoprefixer do a good job with this, but right now, we have to be careful of unnecessarily increasing the size of our CSS for very little RoI
For box-shadow, I don't see a need for prefixing

Reply:
this is useful (y)

Reply:
Agreed with @ankushdharkar 
Auto prefixer should take care of -webkit-box-shadow: and -moz-box-shadow

Reply:
Agreed

Reply:
Even if we have to put it ourselves, I am encouraging you all to check and put only the ones that are really needed. Don't put 4 versions just because you think it would cover the cases. The browsers themselves may not be implementing that variant anymore. Your code has no impact then.
@Shihab-Duzno @tejasberi @Harshi7016